Makefile

```diff
@@ -141,6 +141,11 @@ check-single-includes: build/.ran-cmake
 appimage:
 	bash scripts/genappimage.sh
 
+# Build an appimage with embedded update information
+# nightly-appimage for nightly builds or latest-appimage for a release
+%-appimage:
+	bash scripts/genappimage.sh $*
```

Comment:
any reason not to prefix it instead of suffix? e.g. 

    appimage-%:

it's more intuitive to me.

Reply:
`make nightly-appimage` reads like what it's doing -- making the nightly appimage.  I'm fine either way, but that's why I chose it.